definitions/npm/mongodb_v3.5.x/flow_v0.114.x-/mongodb_v3.5.x.js

```diff
@@ -1,15 +1,16 @@
 // @flow
 
 declare module 'mongodb' {
+  import type { Readable } from 'stream';
```

Comment:
This will import as `any`, see the `CONTRIBUTING.md` for more details.

Reply:
Event with node built ins like this? I have a doubt... :thinking:

Reply:
I've added some tests (I was lazy yesterday). From what I tested it's working properly with node built ins but it should be under `declare module` statement

Reply:
Ah! Sorry, I wasn't paying attention that it's a node core package.